src/modules/auth/auth.routes.js

```diff
@@ -5,4 +5,6 @@ import ValidateAuth from '../../middlewares/ValidateAuth';
 const router = express.Router();
 router.post('/signin', ValidateAuth.validateSigninFormData, AuthController.signin);
 
+router.get('/logout', (req, res) => res.status(200).json({ status: '204', message: 'Successfully Logged out' }));
```

Comment:
@SteveFunso kindly move ```(req, res) => res.status(200).json({ status: '204', message: 'Successfully Logged out' })``` to a controller file and import to the route.

Also, ensure to make use of the `responseHandler` helper methods for consistency with our response

Reply:
ok noted.